scrapers/Masqulin.yml

```
@@ -0,0 +1,24 @@
+name: Masqulin
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - masqulin.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="titlePlayer"]
+      Performers:
+        Name: //span[@class="tour_update_models"]/a
+      Tags:
+        Name: //p[@class="sceneInfo"]/span/a
+      Details:
+        selector: //p[@class="col-lg-6 textDescription"]
```

this needs some pp to remove the extra `...` and the `Read more`
```suggestion
      Details:
        selector: //p[@class="col-lg-6 textDescription"]
        postProcess:
          - replace:
              - regex: \.\.\.
                with:
              - regex: \s*Read\smore\s*$
```